ha-example/minio-install/README.md

```diff
@@ -0,0 +1,43 @@
+# minio installation
+
+This folder contains an example minio deployment yaml and the instructions are based on the [minio documentation](https://min.io/docs/minio/kubernetes/upstream/index.html).
+
+1. Deploy the minio with default configurations
+   ```
+   kubectl apply -f minio.yaml -n flink
+   ```
+2. Access the MinIO S3 Console
+   ```
+   kubectl port-forward deployment/minio 9000 9090 -n flink
+   ```
+3. Create a bucket via MinIO S3 Console
+
+   Open `http://localhost:9090`, and login with `minioadmin/minioadmin`, then go to `Buckets` -> `Create Bucket` to create a bucket.
+4. Monitor the files in the bucket
```

Comment:
Wouldn't be better to provide cli commands for that? There is `mc` command in minio container. This is just an idea, I have no strong opinion.

Reply:
Because we are running the examples manually, I think it'd be easier to check via UI, not CLI.